rhp/v3/rhp.go

```diff
@@ -307,6 +307,13 @@ func (pt *HostPriceTable) HasSectorCost() ResourceCost {
 	}
 }
 
+// InitProgramCost is the cost of initialising an mdm program.
+func (pt *HostPriceTable) InitProgramCost() ResourceCost {
```

Comment:
how about `BaseCost`?

Reply:
Ok, I updated it to `BaseCost`. Honestly any name is fine with me. I just want to get rid of 
```go
rc := pt.ReadSectorCost(...)
cost, _ := rc.Total()
cost = cost.Add(pt.InitBaseCost)
```